rescan.go

```diff
@@ -120,6 +125,18 @@ func NotificationHandlers(ntfn rpcclient.NotificationHandlers) RescanOption {
 	}
 }
 
+// ProgressHandler specified a handler to be used when the utxo
+// scanner reports its progress.
+// The passed handler should be non-blocking for the rescan to continue
+// normally.
+func ProgressHandler(
+	handler scanProgressHandler) RescanOption {
```

Comment:
Since `ProgressHandler` is exported, we should also export `scanProgressHandler`.

Reply:
Nice catch! fixed.